content/guides/analog-discovery/_index.md

```diff
@@ -0,0 +1,886 @@
+---
+title: 'Interacting with projects and making measurements with the Analog Discovery'
+linkTitle: Testing Projects with the Analog Discovery 
+description: 'Using the Digilent Analog Discovery 3 to control and probe projects'
+weight: 60
+---
+
+
+This is an example of using the [Digilent Analog Discovery 3](https://digilent.com/reference/test-and-measurement/analog-discovery-3/start) multi-function test and measurement device to characterize a project on the [Tiny Tapeout 05 ASIC](/runs/tt05/), in particular [#271: Analog Testmacro](https://tinytapeout.com/runs/tt05/tt_um_tt05_analog_test), which is a test analog circuit included on TT05 containing both a ring oscillator and a 3-bit DAC.
+
+
+There is a sample *WaveForms* workspace that is ready to use that can 
+   * talk to the demoboard REPL over serial to setup the correct project;
+   * drive the demoboard inputs to dynamically control the project;
+   * monitor and measure the ring oscillator and DAC outputs; and
+   * graph the results and output CSV files with data.
+
+
+The purpose here is to walk-through the entire process while introducing various features of the AD3 and some of the ways to use it.  If you just want the bullet point list of steps without any details, skip to the end of this page.
+
+
+### WaveForms Installation
+
+To play along, an Analog Discovery 3 is required, as is its front-end application, "Waveforms".  Follow the [AD3 Getting Started Guide](https://digilent.com/reference/test-and-measurement/analog-discovery-3/getting-started#download_and_install_waveforms) to get everything setup and test that it is working.
+
+
+### Running The Full Test Suite
+
+We'll walk through every important step, below, and that is a good way to get to know the Analog Discovery.  However, if you want to just get going and explore things on your own, you may simply wire things up and run the pre-built workspace on the AD3, you may just follow the directions in this section.
```

Comment:
end of sentence is a bit clunky. how about:

"you can wire things up, load the provided workspace on the AD3 and follow the bullet point list of steps"

also that uses the same language as above so the user is more sure of the 2 options. bullet point or full walkthrough